src/style.css

```diff
@@ -1,3 +1,72 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+  background-color: #fff;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+}
+
+.list {
+  list-style: none;
+  display: flex;
+  align-items: center;
+}
+
+._link {
+  display: flex;
+  height: 60px;
+  align-items: center;
+  text-decoration: none;
+  margin: 0 10px;
+  color: black;
+  text-transform: uppercase;
+}
+
+._link:hover {
+  color: #00acdc;
+}
+
+.list-item:first-child ._link {
+  margin-left: 0;
+}
+
+.list-item:last-child ._link {
+  margin-right: 0;
+}
+
+.is-active {
+  color: #00acdc;
```

Comment:
[checklist](https://github.com/mate-academy/layout_moyo-header?tab=readme-ov-file#checklist): CSS Variable is used for a blue color